semantic_conventions.md

```diff
@@ -25,13 +25,15 @@ Span tags apply to **the entire Span**; as such, they apply to the entire timera
 | `http.method` | string | HTTP method of the request for the associated Span. E.g., `"GET"`, `"POST"` |
 | `http.status_code` | integer | HTTP response status code for the associated Span. E.g., 200, 503, 404 |
 | `http.url` | string | URL of the request being handled in this segment of the trace, in standard URI format. E.g., `"https://domain.net/path/to?resource=here"` |
+| `mom.destination` | string | An address at which messages can be exchanged. E.g. A Kafka record has an associated `"topic name"` that can be extracted by the instrumented producer or consumer and stored using this tag. |
```

Comment:
To Ben's point, if I hadn't read the name of the pull request I wouldn't know what the acronym meant